package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "frontend-starter-app",
+  "name": "provisioning-frontend",
```

Comment:
Is this intentional? Does not look like a npm only update.

Reply:
That will be on me I think ... I did update package.json, but did not regenerate the lock, so that change is due from my change :(